FernandoAscencio/theia
# [vscode] Error message with vscode.open command

#### What it does
Adds user feedback when attempting to open a file that does not exist.
Partial implementation of 6568.
Closes 5667

#### How to test
1. Install [this](https://github.com/tolusha/vscode-test-extensions/blob/master/open-some-file-0.0.1.vsix) extension. As provided in 6568, made by tolusha
2. Run the `Open Some File` command.
3. Verify the error message appears.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
